task/Taskfile.lagoon.yml

```diff
@@ -0,0 +1,100 @@
+# This is a go-task file solely for Lagoon integration.
+# It can be used independently but is meant to be used
+# in combination with the go-task file in the root of the project.
```

Comment:
Stating that this can be used independently is not correct. This file uses tasks provided by the root `Taskfile.yml` where it also included. The fact that I split it out is primarily a matter of keeping taskfiles somewhat smaller and more focused.

Is there a better way to do this?

Reply:
I think the way it has been done here is fine.
In my opinion go-task has limitations working with included Taskfiles and task cmd scopes.
I like the idea of separating chunks of functionality into separate files.
I have been struggling  with task calling other tasks belonging to another file as well.

Reply:
:+1: I updated the description a bit.